<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapInfoEditor</class>
 <widget class="QDialog" name="MapInfoEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MapInfoEditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Description</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Suggested Players:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="suggestedPlayers"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="description"/>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="author"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Map Version:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="mapVersion">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Editor Version:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="editorVersion">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Loading Screen</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Loading Screen Graphic</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="0" column="0">
            <widget class="QRadioButton" name="useDefaultLoadingScreen">
             <property name="text">
              <string>Use Default Screen</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">loadingScreenGraphicGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="useCampaignLoadingScreen">
             <property name="text">
              <string>Use Campaign Screen</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">loadingScreenGraphicGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="useImportedLoadingScreen">
             <property name="text">
              <string>Use Imported File</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">loadingScreenGraphicGroup</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="campaignLoadingScreen">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxVisibleItems">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="importedLoadingScreen">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxVisibleItems">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Loading Screen Title</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="loadingScreenTitle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Loading Screen Subtitle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="loadingScreenSubtitle"/>
         </item>
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Loading Screen Text</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="loadingScreenText"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Options</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="meleeMap">
           <property name="text">
            <string>Melee Map</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="hideMinimapPreview">
           <property name="text">
            <string>Hide minimap in preview screens</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="maskedPartiallyVisible">
           <property name="text">
            <string>Masked areas are partially visible</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cliffWaves">
           <property name="text">
            <string>Show water waves on cliff shores</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="rollingShoreWaves">
           <property name="text">
            <string>Show water waves on rolling shores</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="itemClassification">
           <property name="text">
            <string>Use item classification system</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="terrainFogBox">
           <property name="title">
            <string>Use Terrain Fog</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Style</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="fogStyle">
              <item>
               <property name="text">
                <string>Linear</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Exponential 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Exponential 2</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Z Start</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDoubleSpinBox" name="fogZStart">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Z End</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDoubleSpinBox" name="fogZEnd">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>100000.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Density</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="fogDensity">
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <double>100.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="ColorButton" name="fogColor">
              <property name="text">
               <string>Choose Color</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="globalWeather">
             <property name="text">
              <string>Use Global Weather</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="globalWeatherCombo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxVisibleItems">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="customSound">
             <property name="text">
              <string>Custom Sound Environment</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="customSoundCombo">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="customLighting">
             <property name="text">
              <string>Custom Light Environment</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="customLightingCombo">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maxVisibleItems">
              <number>20</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Game Data Set:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="gameDataSet">
             <item>
              <property name="text">
               <string>Default (Map melee status)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Custom (TFT 1.02, RoC 1.01)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Melee (Latest Patch)</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="waterTinting">
             <property name="text">
              <string>Use Water Tinting Color</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="ColorButton" name="waterColor">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Choose Color</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Preferences</string>
        </attribute>
        <widget class="QLabel" name="label_17">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>170</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>?</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QPushButton</extends>
   <header>ColorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useImportedLoadingScreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>importedLoadingScreen</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useCampaignLoadingScreen</sender>
   <signal>toggled(bool)</signal>
   <receiver>campaignLoadingScreen</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>globalWeather</sender>
   <signal>toggled(bool)</signal>
   <receiver>globalWeatherCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customSound</sender>
   <signal>toggled(bool)</signal>
   <receiver>customSoundCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customLighting</sender>
   <signal>toggled(bool)</signal>
   <receiver>customLightingCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>waterTinting</sender>
   <signal>toggled(bool)</signal>
   <receiver>waterColor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>435</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="loadingScreenGraphicGroup"/>
 </buttongroups>
</ui>